cli/cmd/vm_rm.go

```diff
@@ -36,12 +35,13 @@ replicated vm rm --all
 # Perform a dry run of removing all VMs
 replicated vm rm --all --dry-run`,
 		RunE:              r.removeVMs,
-		ValidArgsFunction: r.completeVMIDs,
+		ValidArgsFunction: r.completeVMIDsAndNames,
```

Comment:
The `rm` functionality across these commands can complete VMs, networks or clusters that are outside of `"running"` state, so that needs to be fixed.